Apps/WebClient/src/ClientApp/src/router.ts

```diff
@@ -238,7 +238,7 @@ const routes = [
         path: "/login",
         component: LoginView,
         props: (route: Route) => ({
-            isRetry: route.query.isRetry,
+            isRetry: route.query.isRetry === "true",
```

Comment:
What's this?

Reply:
passes the property as a boolean instead of a string

Reply:
passes the property as a boolean instead of a string